requirements.txt

```diff
@@ -8,9 +8,9 @@ sphinx_rtd_theme
 mpld3
 bokeh==0.12.14
 smac>=0.8.0
+configspace>=0.4.6
```

Comment:
What changed from 0.4.2 to 0.4.6 that this version increase is necessary?

Reply:
I don't know, but it stopped throwing errors after updating. Should I investigate?